docs/Config.md

```diff
@@ -86,6 +86,7 @@ gui:
   border: 'rounded' # one of 'single' | 'double' | 'rounded' | 'hidden'
   animateExplosion: true # shows an explosion animation when nuking the working tree
   portraitMode: 'auto' # one of 'auto' | 'never' | 'always'
+  filterMode: 'substring' # one of 'substring' | 'fuzzy'
```

Comment:
I would document exactly how substring works (further down in this file in its own section) because it won't be obvious to users

Reply:
Added a section in 010ef7dd055b. You are better with words than me, feel free to improve it.